components/profile/radio-group.vue

```diff
@@ -0,0 +1,71 @@
+<template>
+  <div role="group" class="switches with-labels">
+    <label
+      v-for="item in items"
+      :key="item.id"
+      class="clickable"
+    >
+      <span class="label">{{ item.label }}</span>
+      <div class="radio">
```

Comment:
Can't the RadioGroup UI component be reused here somehow?
Even if it will require a bit of refactoring, it should be done because the point of UI components are to be as reusable as possible

Reply:
The logic of this radio group is not yet fully done. It'll be much bigger and more difficult to make it reusable. Don't like to too, but have to live with that